src/DataTransformer/Wishlist/WishlistTokenValueAwareInputCommandDataTransformer.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusWishlistPlugin\DataTransformer\Wishlist;
+
+use ApiPlatform\Core\Serializer\AbstractItemNormalizer;
+use BitBag\SyliusWishlistPlugin\Command\Wishlist\WishlistTokenValueAwareInterface;
+use BitBag\SyliusWishlistPlugin\Entity\WishlistInterface;
+use Sylius\Bundle\ApiBundle\DataTransformer\CommandDataTransformerInterface;
+
+class WishlistTokenValueAwareInputCommandDataTransformer implements CommandDataTransformerInterface
+{
+    public function transform($object, string $to, array $context = []): WishlistTokenValueAwareInterface
+    {
+        /** @var WishlistInterface $wishList */
+        $wishList = $context[AbstractItemNormalizer::OBJECT_TO_POPULATE];
+
+        $object->setWishlistTokenValue($wishList->getToken());
```

Comment:
@pptasinski can we pass `$whishlist` object to command here as a shortcut to skip fetching the same entity in `commandHandler` via repository method `findBy` etc. ?